### PR TITLE
Additional context and code updates to spatial-data tutorial

### PR DESCRIPTION
**Pull request by mjones01**
Major updates: 

* updated packages & code for mapping w/ ggplot & using spatialDataPolygons
* updated IS portion to work with the sensor positions file that is now available through neonUtilities
* throughout added additional context and how users find the information to get to the next step
* changes a few style elements to match updated tutorial template 
* added additional H2 headers to make it easier to find sections in webpage navigation

@cklunch - please review.  I'm hoping to have this live by end of day tomorrow if you have time.

### CODE REVIEW

File: tutorials/R/spatial-data/spatialData.Rmd [hunk 1]
```diff
@@ -22,331 +23,514 @@ connecting remote sensing with ground-based measurements, the methods in
...
-gg <- ggplot() + theme_map()
-gg <- gg + geom_map(data=map, map=map,
-                    aes(x=long, y=lat, map_id=id, group=group),
-                    fill="white", color="black", size=0.3)
-gg
+```{r plot-domains, fig.width=8, fig.height=6}
+# plot domains
+domainMap <- ggplot(neonDomainsDF) + 
+        geom_map(map = neonDomainsDF,
+                aes(x = long, y = lat, map_id = id),
```
Comment (mjones01) [inline tutorials/R/spatial-data/spatialData.Rmd:173]: This is an area I'm not 100% happy with.  As written get warning "Warning message: Ignoring unknown aesthetics: x, y ", in addition geom_map says nothing about including x & y.  But if not included, plot doesn't work.   Tried to find a solution for too long so just leaving as is for now.

File: tutorials/R/spatial-data/spatialData.Rmd [hunk 2]
```diff
@@ -22,331 +23,514 @@ connecting remote sensing with ground-based measurements, the methods in
...
+# set working directory to ensure R can find the file we wish to import and where
+# we want to save our files. 
+
+wd <- "~/Documents/data/" # This will depend on your local environment
+setwd(wd)
+
 ```
 
-## Spatial data files
+## Source 1: NEON spatial data files
```
Comment (mjones01) [inline tutorials/R/spatial-data/spatialData.Rmd:84]: Is this confusing?  In order to have an H2 header for each of the sections where we do different things to allow for easier in page navigation, I made these "super" H2 headers by calling out the data sources.

File: tutorials/R/spatial-data/spatialData.Rmd [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+description: "Use files available on the NEON data portal, NEON API, and  neonUtilities R package to access the locations of NEON sampling events and infrastructure. Calculate more precise locations for certain sampling types and reference ground sampling to airborne data."
 dateCreated: 2019-09-13
 authors: [Claire K. Lunch]
-contributors:
+contributors: Megan Jones
 estimatedTime: 40 minutes
 packagesLibraries: neonUtilities
 topics: data-management, rep-sci
 languageTool: R
+dataProduct:
```
Comment (mjones01) [inline tutorials/R/spatial-data/spatialData.Rmd:12]: Added this back in as we're considering if a "all" is appropriate for mapping to data product pages for those tutorials that apply to all data products.

File: tutorials/R/spatial-data/spatialData.Rmd [hunk 4]
```diff
@@ -22,331 +23,514 @@ connecting remote sensing with ground-based measurements, the methods in
...
-### Observational data
+The second source of data is spatial data in and accompanying data directly 
+downloaded from the NEON data portal. How the data are handled are a bit different 
+for the Observational Sampling data and the Instrumented Systems data, so we'll 
+work through working with both types. 
 
+## Locations for observational data
+
+### Plot level locations
 Both aquatic and terrestrial observational data downloads include spatial
```
Comment (mjones01) [inline tutorials/R/spatial-data/spatialData.Rmd:275]: There is one minor mention of aquatics sites here but it is pretty much ignored throughout the rest of the tutorial.  Not this iteration but I think we should consider adding in something about location data for aquatic data products.

File: tutorials/R/spatial-data/spatialData.Rmd (hunk 1 above)
Comment (cklunch) [reply tutorials/R/spatial-data/spatialData.Rmd:173]: Yes, I had the same problem when I set it up in the first place, it's totally maddening! I never found a solution either.

File: tutorials/R/spatial-data/spatialData.Rmd (hunk 2 above)
Comment (cklunch) [reply tutorials/R/spatial-data/spatialData.Rmd:84]: Looks good to me!

File: tutorials/R/spatial-data/spatialData.Rmd (hunk 4 above)
Comment (cklunch) [reply tutorials/R/spatial-data/spatialData.Rmd:275]: Agreed. I haven't really explored the bathymetry/geomorphology products, but there's probably something cool that could be done there. Could potentially combine with AOP data, too.

**Comment by cklunch** [comment]
Thanks for adding so much great explanatory text! Your changes look great to me.

**Pull request merged**